### PR TITLE
add token address to search

### PR DESCRIPTION
**Pull request by skylarbarrera**
fixes RNBW-1156

PoW: http://cloud.skylarbarrera.com/Screen-Recording-2021-05-18-13-02-58.mp4

### CODE REVIEW

**Review by mrbaguvix — APPROVED** [review]
Looks good

File: src/components/coin-row/ExchangeCoinRow.js [hunk 1]
```diff
@@ -38,8 +39,13 @@ const FloatingFavoriteEmojis = styled(FloatingEmojis).attrs({
   z-index: 100;
 `;
 
-const BottomRow = ({ showBalance, symbol }) =>
-  showBalance ? null : <BottomRowText>{symbol}</BottomRowText>;
+const BottomRow = ({ showBalance, symbol, address }) => {
```
Comment (jinchung) [inline src/components/coin-row/ExchangeCoinRow.js:42]: alphabetize (address first)

**Comment by jinchung** [comment]
@skylarbarrera is this the PR where you'll move it out into a tooltip instead?

**Comment by skylarbarrera** [comment]
> @skylarbarrera is this the PR where you'll move it out into a tooltip instead?

that's the plan, will tackle this once I finish cms setup

**Pull request closed without merging**